### ISSUE TITLE
training resnet3d

### ISSUE THREAD

**Issue by YuanLiuuuuuu** [neuroailab/VIE#16]
Hi, have you ever tried training resnet3d model with only IR loss. Now I am implementing the pytorch version, however I find that after training for 250000 steps with IR loss, I can only reach top1_10nn about 20%. Is it necessary to use LA loss in order to get the best performance?

**Comment by chengxuz** [comment]
That performance may not be too low. I have not trained a VIE-IR for 3dresnet18, but according to my experience on training IR and LA models for other architectures, if you are using the same input resolution and the same architecture as the results in https://github.com/neuroailab/VIE/blob/master/notebook/show_results.ipynb, then the performance of VIE-IR should be lower by 2 to 3 percents to VIE-LA, so 20% is not super low (you should compare to the number corresponding to 50 in the x-axis).

**Comment by YuanLiuuuuuu** [comment]
which step is the x axis in https://github.com/neuroailab/VIE/blob/master/notebook/show_results.ipynb corresponding to?

**Comment by chengxuz** [comment]
X-axis represents the number of validations done, so it's every 5000 steps if you are using the same setting. For your 250,000 steps result, it should correspond to 50 in x.

**Comment by YuanLiuuuuuu** [comment]
Hi, I have anther question, in your training of 3dresnet model, did you save the memory bank after you finished training the model with IR loss and load the previous saved memory bank for training the model with LA loss? I reimplemented your code with pytorch, I found that when I resumed training the model with LA loss, which had already been trained with IR for 50000 steps, there is a drop in accuracy.

**Comment by chengxuz** [comment]
Yes, I did save the memory bank and restore from the memory bank.

**Comment by YuanLiuuuuuu** [comment]
Hi, do you have log about training transfer-resnet3d model on KN. After training for 140000 steps, I can only reach conv3:0.13298434019088745, conv4:0.24137848615646362, conv5:0.34119871258735657. How many steps did you traine to get these results in the paper?

**Comment by chengxuz** [comment]
I have added the logs for both ImageNet and Kinetics transfer learning to https://github.com/neuroailab/VIE/blob/master/notebook/show_results.ipynb, hope it helps! It will definitely require more than 140,000 steps to get the results, I believe the Kinetics transfer learning exp I did took like 120*5000 steps (as you can see from the logs).

**Comment by YuanLiuuuuuu** [comment]
Hi, when resume training with tfutils with multiple gpus, your code will automatically load all parameters in your checkpoint. However, I only want to load part of these parameters, for example, I would not like to import  the memory bank, how can I achieve this? Thank you!
